RealEstate/apps/core/views.py

```diff
@@ -45,3 +46,83 @@ def get(self, request, *args, **kwargs):
+
+        # Merging grades and categories to provide object with both information.
+        # Data Structure: [(cat1, score1), (cat2, score2), ...]
+        graded = []
+        for category in categories:
+            missing = True
+            for grade in grades:
+                if grade.category.id is category.id:
+                    graded.append((category, grade.score))
+                    missing = False
+                    break
+            if missing:
+                graded.append((category, None))
+                
+        class ContactForm(forms.Form):
+            def __init__(self, *args, **kwargs):
+                super(ContactForm, self).__init__(*args, **kwargs)
+                for c, s in graded:
+                    self.fields[c.summary] = forms.CharField(initial="3" if None else s, widget=forms.HiddenInput())
+
+        context = {'couple': couple, 'house' : house.first(), 'grades': graded, "form" : ContactForm() }
+        return render(request, 'core/houseEval.html', context)
+
+    def post(self, request, *args, **kwargs):
+        """
+        Depending on what functionality we want, the post may be more of a redirect back to the home page. In that
+        case, much of this code will leave. In the meantime, it saves new data, recreates the same form and posts a
+        success message.
+        """
+        homebuyer = Homebuyer.objects.filter(user_id=request.user.id)
+        couple = Couple.objects.filter(homebuyer__user=request.user)
+        categories = Category.objects.filter(couple=couple)
+        house = House.objects.filter(couple=couple)
+
+        for category in categories:
+            value = request.POST.get(category.summary)
+            if not value:
+              value = 3
+            grade, created = Grade.objects.update_or_create(
+                homebuyer=homebuyer.first(), category=category, house=house.first(), defaults={'score': int(value)})
+
+        grades = Grade.objects.filter(house=house.first(), homebuyer=homebuyer)
+
+        # Merging grades and categories to provide object with both information.
+        # Data Structure: [(cat1, score1), (cat2, score2), ...]
+        graded = []
+        for category in categories:
+            missing = True
+            for grade in grades:
+                if grade.category.id is category.id:
+                    graded.append((category, grade.score))
+                    missing = False
+                    break
+            if missing:
+                graded.append((category, None))
+
+        class ContactForm(forms.Form):
+            def __init__(self, *args, **kwargs):
+                super(ContactForm, self).__init__(*args, **kwargs)
+                for c, s in graded:
+                    self.fields[c.summary] = forms.CharField(initial="0" if None else s, widget=forms.HiddenInput())
+        
+        context = {'couple': couple, 'house' : house[0], 'grades': graded, "form" : ContactForm(), "success": True }
```

Comment:
Django has a mechanism for flashing one-off messages.

from django.contrib import messages
messages.success(request, 'message here')

Then in the template:
{% boostrap_messages %}

This would be a replacement for the 'success' and bootstrap_alert.